front/global.types.d.ts

```diff
@@ -1,2 +1,3 @@
 declare module '*.png';
 declare module '*.svg';
+declare module '*.css';
```

Comment:
This is not necessary, we don't have any css file

Reply:
Done